src/matlab/IKServer.m

```diff
@@ -26,7 +27,7 @@
     function obj = loadNominalData(obj,filename)
       if nargin < 1
         filename = [getenv('DRC_BASE'), ...
-          '/software/control/matlab/data/atlas_bdi_fp.mat'];
+          '/software/control/matlab/data/atlas_v3/atlas_bdi_fp.mat'];
```

Comment:
I agree and obviously v3 hasn't been used in a long time.

Note that I reorganised the folder structure of the fixed points. Would you agree with moving the fixed points out of the control folder and into the specific robot model directories?